### PR TITLE
Add configuration support for QA cloud environment

### PR DESCRIPTION
**Pull request by joseph-flinn**
## Summary

We need to expand the configurations of the backend services to allow for hosting the QA environment along side Production while acting as much like Production as possible.

The ASPNETCORE_ENVIRONMENT AppSetting in the Azure App Services has been updated in the QA environment to be "QA"

### CODE REVIEW

**Comment by kspearrin** [comment]
Are you going to remove some of this logging after testing? I dont think we want that in there for production.

File: src/Billing/Controllers/StripeController.cs [hunk 1]
```diff
@@ -104,6 +104,10 @@ public async Task<IActionResult> PostWebhook([FromQuery] string key)
                 _logger.LogWarning("Getting test events in production.");
                 return new BadRequestResult();
             }
+            else if (_hostingEnvironment.IsEnvironment("QA"))
+            {
+                _logger.LogWarning("Getting events in QA.");
```
Comment (cscharf) [inline src/Billing/Controllers/StripeController.cs:109]: Seems like if we're not expecting events in QA we should return a `new BadRequestResult()` here as well. OR, if we're getting `live` events in QA, etc. This feels wrong logging a warning w/o taking some appropriate action

File: src/Billing/Controllers/PayPalController.cs [hunk 2]
```diff
@@ -48,6 +48,7 @@ public class PayPalController : Controller
         [HttpPost("ipn")]
         public async Task<IActionResult> PostIpn()
         {
+            _logger.LogWarning("PayPal webhook has been hit.");
```
Comment (cscharf) [inline src/Billing/Controllers/PayPalController.cs]: Do we need these logs here, especially as a `WARN` level log (seems like a `DEBUG` level or `INFO` at most).

**Comment by joseph-flinn** [comment]
I changed the first logging statement in the PayPal Controller to be at the Information level. Production and QA are set up to ignore the Information level logging. I was planning on keeping them in there as is, but I could remove them.

File: src/Billing/Controllers/StripeController.cs (hunk 1 above)
Comment (joseph-flinn) [reply src/Billing/Controllers/StripeController.cs]: oh, yep. This was debugging code that I forgot to remove for those emails from Stripe over the weekend. This has been removed.

**Review by cscharf — APPROVED** [review]

**Pull request merged**